### DISCUSSION TITLE
More than one search box possible?

### DISCUSSION THREAD

**Discussion by spencersouv** [microsoft-search/pnp-modern-search#2399] [Q&A] [answered]
I have a search box on a homepage that takes me to a search results page.
On that search results page, I also have another search box.

Is it possible to associate the search results web part with more than one search box?
Search box on same page and another search box on separate page.

**Comment by wobba** [comment, marked as answer]
1. Page one sends you to page two with `q=foo`
2. Configure search box on page two to consume the `q=` parameter via dynamic data
3. Configure the result web part to get the query from your search box web part via dynamic data

**Reply by spencersouv** [reply]
Thank you so much!
